docs/.vuepress/head.js

```diff
@@ -14,7 +14,7 @@ module.exports = [
     document.addEventListener("DOMContentLoaded", function() {
       var banner = document.createElement("div");
       var text1a = document.createElement("span");
-      text1a.innerHTML = "If you have signed up for Esper on or after 12th April, 2022, please click ";
+      text1a.innerHTML = "If you have signed up for Esper using out website- https://www.esper.io/, please click ";
```

Comment:
Revise to "If you signed up for an Esper account through our website [hyperlink https://www.esper.io/ to website], please click ..."